examples/lfric/eg14/README.md

```diff
@@ -31,12 +31,12 @@ The binary can be executed using ``example_openacc`` without additional paramete
 ./example_openacc
  Mesh has           5 layers.
  profile_PSyDataInit called
- PreStart called for module 'main_psy' region 'invoke_initialise_fields:r0'
- PostEnd called for module 'main_psy' region 'invoke_initialise_fields:r0'
+ PreStart called for module 'main_psy' region 'invoke_initialise_fields-r0'
```

Comment:
In your comment earlier, you said:

"Note that this patch only affects the : in region names, NOT any colon added by e.g. psydata libraries when using the module- and region-name for creating region names (which are fine, since they are not creating files). I think it is consistent that we will leave it to the various psydata apps how to name things (and using : actually makes it possible to see what is region and what module ... before we had sometimes three :, meaning it was hard to see what was module and region (though de-facto it should be obvious). Now we have : between region and module, and - between the parts inside a region"

but I don't see any colons anywhere now?

Reply:
I realise that this change is partly (mostly?) my fault but, now that I look at the output, I think the colons made the profiling-region *names* (i.e. not affecting any filenames) more readable. What do you think?

Reply:
I agree, and the colons are still in all libraries except template, since template just prints then individual. E.g.:
Using template-lib (which prints them individually):
```
~/work/psyclone/examples/gocean/eg5/profile$ ./profile_test.template 
 profile_PSyDataInit called
...
 PreStart called for module 'invoke_0' region 'r0'
 PostEnd called for module 'invoke_0' region 'r0'
 PreStart called for module 'invoke_1_update_field' region 'r0'
 PostEnd called for module 'invoke_1_update_field' region 'r0'
```
Using simple_timer:
```
~/work/psyclone/examples/gocean/eg5/profile$ ./profile_test.simple_timing 
 ===========================================
 module::region            		count		sum			min		average			max
 invoke_0::r0                        1 	   0.00000000     	   0.00000000     	   0.00000000     	   0.00000000    
 invoke_1_update_field::r0           1 	   0.00000000     	   0.00000000     	   0.00000000     	   0.00000000    
 ===========================================
```
For reasons that I can't remember it uses a `::`

dl_timer:
```
~/work/psyclone/examples/gocean/eg5/profile$ ./profile_test.dl_timer 
-----------------------------------------------------------------------------
Region                          Counts     Total       Average*     Std Err
-----------------------------------------------------------------------------
Thread   0
invoke_0:r0                          1  0.25590E-05   0.23971E-05  0.00E+00
invoke_1_update_field:r0             1  0.19640E-05   0.18021E-05  0.00E+00
-----------------------------------------------------------------------------
```
Unchanged a single `:` 

All of these examples appear to be affected by #2641, the module name should not be `invoke_`.

So, the profile module and region separator is still `"`, only there are no more `:` within a region name.

Reply:
Great, thanks Joerg.